README.md

```diff
@@ -20,3 +20,48 @@ This is a complete reboot of some earlier ideas that began in a Backbone app. If
 * Tolu 'The Rock' Komolafe;
 * Jesmine 'Ghost Rider' Omonori; and,
 * Bart 'Kick-Ass' Riordan.
+
+
+## Releasing:
+Right now, until we get some gulp scripts in place make sure to run the following before packaging and releasing a version:
+* `npm run build`
+* `npm run build-min`
+
+## What's included: 
+
+* Action Prototype
+Based of off facebook patterns and intended to be used with flux. It works with a loose $.ajax wrapper to fire
+API calls to your server. It supplies a `fireApi` function to the action, so that you can use it when you define your
+own actions. You specific a REST actions (get, post, etc), and endpoint, and optional data payload (for a put or a post). You can optionally supply action types to dispatch based on a success or failure from the server. Here's an example of how to use:
+
+```
+//in MyActions.js
+var TestAction = assign(ActionPrototype, {
+  Types: KeyMirror({
+    GOT_USERS: null
+  }),
+  fetchUsers: function() {
+     this.fireApi('get', 'assessment', null,
+      {successAction: this.Types.GOT_ASK_DOCUMENT_GROUPS, //Note that since no failureAction was specified, it will dispatch this action always. 
+      JSONHead: 'assessment'}) //head of the json coming back from the server. 
```

Comment:
@BJK Because Python style guides generally recommend it I still prefer two spaces before the slashes indicating inline comments and one space between the slashes and the actual comment. Thoughts? I can add it to our JavaScript style guide if we can settle on a preference.